### PR TITLE
Added page routes

### PR DESCRIPTION
**Pull request by mwangimuringi**
export default function App() {
  return (
    <BrowserRouter>
      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="/sign-in" element={<SignIn />} />
        <Route path="/sign-up" element={<SignUp />} />
        <Route path="/about" element={<About />} />
        <Route path="/profile" element={<Profile />} />
      </Routes>
    </BrowserRouter>
  )
}

### CODE REVIEW

File: README.md
Comment (muringitech) [inline README.md:1]: this is ok

File: client/package.json
Comment (muringitech) [inline client/package.json:1]: awesome

File: client/src/App.jsx
Comment (muringitech) [inline client/src/App.jsx:1]: Ok in adding routes

File: client/src/pages/About.jsx
Comment (muringitech) [inline client/src/pages/About.jsx:1]: ok

File: client/src/pages/Home.jsx
Comment (muringitech) [inline client/src/pages/Home.jsx:1]: ok

**Review by muringitech — APPROVED** [review]

File: README.md
Comment (mwangimuringi) [reply README.md:1]: Yeah

File: client/package.json
Comment (mwangimuringi) [reply client/package.json:1]: ok

File: client/src/pages/About.jsx
Comment (mwangimuringi) [reply client/src/pages/About.jsx:1]: Its good

File: client/src/pages/Home.jsx
Comment (mwangimuringi) [reply client/src/pages/Home.jsx:1]: ok

**Pull request merged**